lib/g_searcher_web/controllers/error_handler.ex

```diff
@@ -10,4 +12,20 @@ defmodule GSearcherWeb.ErrorHandler do
     |> render("#{status}_page.html")
     |> halt()
   end
+
+  def combine_changeset_errors(changeset) do
```

Comment:
How about rename this function to be more related to building error messages? I confuse read function name at first.